get.js

```diff
@@ -31,7 +31,17 @@ if (!fs.existsSync(hbwgConfig.tempDir)) {
     !fs.existsSync(`${hbwgConfig.tempDir}/.version.hbwg_cache`) ||
     fs.readFileSync(`${hbwgConfig.tempDir}/.version.hbwg_cache`) != VERSION
   ) {
-    ChildProcess.execSync(`rm ${hbwgConfig.tempDir}/ -rf`);
+    try {
+      fs.rmSync(hbwgConfig.tempDir, { recursive: true, force: true });
+      logback("Temporary directory deleted successfully");
```

Comment:
please remove try...catch... and all log function use. user can get the wrong infomation from nodejs.